src/azure-cli-core/azure/cli/core/_msal.py

```diff
@@ -0,0 +1,30 @@
+from knack.util import CLIError
+from msal import ClientApplication
+
+
+class SSHCertificateClientApplication(ClientApplication):
+    """
+    This class is added only for vmssh feature.
+    This is a temporary solution and will deprecate after adoption to MSAL completely.
+    """
+    def _acquire_token_silent_by_finding_rt_belongs_to_me_or_my_family(
```

Comment:
seems account not used, how about remove this wrapper?

Reply:
It's a method which defined in the parent class. We'd better not to change the method signature  in subclasses.